DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -346,7 +346,7 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
     }
 
     private func refreshDataVolumeTotals() async {
-        guard let activeSession = try? await ConnectionSessionUtilities.activeSession() else {
+        guard let activeSession = await tunnelController.activeSession() else {
```

Comment:
This is the core change of this PR, we now reuse the active session from the tunnel controller. Because we use the tunnel controller from the app dependency provider, it means we have one session reused throughout the entire app.